src/sickle.ts

```diff
@@ -259,17 +266,20 @@ class Annotator {
   }
 
   private writeTextFromOffset(from: number, node: ts.Node): number {
-    let to = node.getFullStart();
-    if (from == to) return to;
-    assert(to > from, 'Offset must not be smaller');
-    return this.writeRange(from, to);
+    return this.writeRange(from, node.getFullStart());
   }
 
-  private fail(msg: string) { throw new Error(msg); }
-}
-
-function assert(condition: boolean, msg: string) {
-  if (!condition) throw new Error(msg);
+  private fail(msg: string) {
```

Comment:
Random drive by review comment: maybe compare with this code here, where we solved the problem previously?

https://github.com/angular/ts2dart/blob/master/lib/main.ts#L241